ipatests/test_integration/test_adtrust_install.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+
+"""This module provides tests for ipa-adtrust-install utility"""
+
+import re
+import textwrap
+
+from ipaplatform.paths import paths
+from ipapython.dn import DN
+from ipatests.pytest_ipa.integration import tasks
+from ipatests.test_integration.base import IntegrationTest
+
+
+class TestIpaAdTrustInstall(IntegrationTest):
+    topology = 'line'
+    num_replicas = 1
+
+    def unconfigure_replica_as_agent(self, host):
```

Comment:
Is this method intended as a "cheap" cleanup instead of topology reinstall? If yes, please add a phrase in docstring, as  it the method looks "hacky" an not obvious.

Reply:
OK, added a comment